HelloPush/README.md

```diff
@@ -0,0 +1,182 @@
+# helloworld-push-android: Basic Mobile Application showing the AeroGear Push feature on Android
+---------
+Author: Daniel Passos (dpassos)  
+Level: Beginner  
+Technologies: Java, Android  
+Summary: A basic example of Push : Registration and receiving messages.  
+Target Product: JBoss Mobile Add-On  
+Product Versions: 1.0.0  
+Source: https://github.com/aerogear/aerogear-push-helloworld/tree/master/android 
+
+## What is it?
+The ```helloworld``` project demonstrates how to include basic push functionality in Android applications.
+
+This simple project consists of a ready-to-build Android application. Before building the application, you must register the Android variant of the application with a running AeroGear UnifiedPush Server instance and Google Cloud Messaging for Android. The resulting unique IDs and other parameters must then be inserted into the application source code. After this is complete, the application can be built and deployed to Android devices.
+
+When the application is deployed to an Android device, the push functionality enables the device to register with the running AeroGear UnifiedPush Server instance and receive push notifications.
+
+## How do I run it?
+
+### 0. System Requirements
+* [Java 7](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Gradle 2.2.1](https://www.gradle.org/downloads)
+* Latest [Android SDK](https://developer.android.com/sdk/index.html)
+* [Platform version 22](http://developer.android.com/tools/revisions/platforms.html)
+* [Build Tools 22.0.1](https://developer.android.com/tools/revisions/build-tools.html)
+* Latest [Android Support Library](http://developer.android.com/tools/support-library/index.html)
```

Comment:
@secondsun Just wondering if is a good idea fix some version here, since it's changing from time to time. wdyt?